Prefix_Sum/Prefix_Sum.js

```diff
@@ -0,0 +1,30 @@
+/*
+PREFIX SUM
+
+The task is to find the prefix sum of every element of a given array.
+Prefix sum equals to the sum of all element from start to the current element of the array.
+
+*/
+
+function prefixSum(array) {
+	const prefix_array = [];
```

Comment:
Fix indentation. Use 4 space for indentation.

Reply:
it is already in 4 space indentation